### PR TITLE
make: Update POT file correctly

### PR DESCRIPTION
**Pull request by VladimirSlavik**
The changes from 70d7814e6eccc1142298e08fb2c18f55c7c15d37 were mistakenly applied to the top level makefile.

----

Needed to fix the daily pot file updates in l10n repo: https://github.com/rhinstaller/anaconda-l10n/actions/workflows/pot-file-update.yaml

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -125,7 +125,10 @@ pot-update-check: pot
 # - copy pot file to this repository
 # - check if pot file is changed (ignore the POT-Creation-Date otherwise it's always changed)
 	@TEMP_DIR=$$(mktemp --tmpdir -d anaconda-localization-XXXXXXXXXX) || exit 5 ; \
-	git clone --depth 1 -b $(GIT_L10N_SHA) -- $(L10N_REPOSITORY) $$TEMP_DIR || exit 6 ; \
```
Comment (M4rtinK) [inline Makefile.am:128]: Please ad a docstring explaining how this works - its too clever for me. :)

**Review by M4rtinK — APPROVED** [review]
Looks good to me, but please add a docstring explaining your magic. :)

File: Makefile.am (hunk 1 above)
Comment (VladimirSlavik) [reply Makefile.am:128]: Done...

**Comment by VladimirSlavik** [comment]
Good that I did not merge it, I just realized the fix was wrong. We want to put the new pot on the branch, not on top of the pinned commit.

**Pull request merged**